root_profiler/sheets/standard/Serenity.php

```diff
@@ -62,7 +62,7 @@
 <!-- Ugly Internet Explorer Hack -->
 <?php
 $agent = $_SERVER['HTTP_USER_AGENT'];
-$firefox = eregi("Firefox", $agent);
+$firefox = preg_match('/Firefox/i', $agent);
 if( $firefox ) { echo '<!--'; } ?>
     <table id="maintable">
       <tr>
```

Comment:
Ideally, we should just get rid of these "Ugly hacks" Internet Explorer is not relevant anymore, and it looks like that hack would affect any chromium-based browsers too.

Reply:
I frankly agree. 10-year-old browser-specific workarounds are going to be utter shite. I was too focused on resolving the conflict to see the deeper issue.